From 043d778ac0dd732427ebe349e7b9d0afc788c952 Mon Sep 17 00:00:00 2001
From: Alan Tull <atull@opensource.altera.com>
Date: Thu, 5 May 2016 14:24:21 -0500
Subject: [PATCH 011/151] FogBugz #365525-3: socfpga: dts: remove fpga image
 specific info

commit  bfd99d01d60b3bb60f5d2e99790af691cbbfffe5 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Remove ranges and fpga-bridges information from the base fpga region.
This should added in the overlay and because it is specific to the
hardware loaded in the FPGA.

Signed-off-by: Alan Tull <atull@opensource.altera.com>
Signed-off-by: Matthew Gerlach <mgerlach@opensource.altera.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm/boot/dts/socfpga.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/socfpga.dtsi b/arch/arm/boot/dts/socfpga.dtsi
index 9a14b5fe3b57..b51a6a9efd09 100644
--- a/arch/arm/boot/dts/socfpga.dtsi
+++ b/arch/arm/boot/dts/socfpga.dtsi
@@ -88,7 +88,7 @@
 			};
 		};
 
-		base_fpga_region {
+		base_fpga_region: base-fpga-region {
 			compatible = "fpga-region";
 			fpga-mgr = <&fpgamgr0>;
 
-- 
2.26.1

